ajax/libs/timelinejs/package.json

```diff
@@ -1,28 +1,30 @@
 {
   "name": "timelinejs",
-  "filename": "js/timeline-min.js",
-  "version": "2.36.0",
+  "filename": "js/timeline.min.js",
```

Comment:
the file `timeline-min.js` is what we need and the minified file of `timeline.js`.